rpc-server/src/config.rs

```diff
@@ -20,8 +20,7 @@ pub struct GenesisInfo {
 impl GenesisInfo {
     pub async fn get(
         near_rpc_client: &crate::utils::JsonRpcClient,
-        s3_client: &near_lake_framework::s3_fetchers::LakeS3Client,
-        s3_bucket_name: &str,
+        fastnear_client: &near_lake_framework::fastnear_client::FastNearClient,
```

Comment:
This change basically indicates disadvantages we are about to introduce to the `near-lake-framework`. I would expect `near_lake_framework::Client` here that could work with either Lake data or FastNear data. We need to rethink the changes on the framework. Though, it is not a blocker for this PR

Reply:
I will improve it on the near-lake-framework side.